src/components/SearchResults.js

```diff
@@ -14,6 +14,20 @@ const SearchBarContainer = styled.div`
 `
 const SearchResults = React.memo(() => {
   let { searchTerm } = useParams()
+  let location = useLocation()
```

Comment:
I've got an idea for this component that may make the logic here a little bit easier to follow. Can we save the location state (basically the query params) as a const like so?
```suggestion
  const { state: parameters } = useLocation();
```

See the use effect below for the next part. In the meantime, we can get rid of `quickCandidateSearchParam` and `quickDonorSearchParam`?

Reply:
I used those to dynamically render 1 table at a time vs both with 1 being empty